offline/QA/Jet/photonjetskinematics.cc

```diff
@@ -0,0 +1,220 @@
+//____________________________________________________________________________..
+
+#include "photonjetskinematics.h"
+#include <fun4all/PHTFileServer.h>
+
+// Fun4all includes
+
+#include <fun4all/Fun4AllHistoManager.h>
+
+#include <fun4all/Fun4AllReturnCodes.h>
+
+#include <phool/PHCompositeNode.h>
+
+#include <TFile.h>
+#include <phool/getClass.h>
+#include "TH1.h"
+#include "TMath.h"
+//#include <jetbase/JetContainer.h>
+
+
+// Tower includes
+#include <calobase/RawCluster.h>
+#include <calobase/RawClusterContainer.h>
+#include <calobase/RawClusterUtility.h>
+#include <calobase/RawTower.h>
+#include <calobase/RawTowerContainer.h>
+#include <calobase/RawTowerGeom.h>
+#include <calobase/RawTowerGeomContainer.h>
+#include <calobase/TowerInfo.h>
+#include <calobase/TowerInfoContainer.h>
+#include <calobase/TowerInfoDefs.h>
+
+// Cluster includes
+#include <calobase/RawCluster.h>
+#include <calobase/RawClusterContainer.h>
+
+// QA includes
+#include <qautils/QAHistManagerDef.h>
+
+// c++ includes
+#include <algorithm>
+#include <cassert>
+#include <iostream>
+#include <vector>
+//____________________________________________________________________________..
+
+photonjetskinematics::photonjetskinematics(const std::string &m_modulename, const std::string &m_inputnode) :
+  SubsysReco(m_modulename)
+  , modulename(m_modulename)
+  , inputnode(m_inputnode)
+  , histtag("AllTrig")
+  , trgToSelect(JetQADefs::GL1::MBDNSJet1)
+  , doTrgSelect(false)
```

Comment:
Here we'll need to initialize `manager`, so something like so:
```suggestion
  SubsysReco(m_modulename)
  , modulename(m_modulename)
  , inputnode(m_inputnode)
  , histtag("AllTrig")
  , trgToSelect(JetQADefs::GL1::MBDNSJet1)
  , doTrgSelect(false)
  , manager(nullptr)
```